src/overlays/actors/ovl_En_Bji_01/z_en_bji_01.c

```diff
@@ -21,36 +44,405 @@ const ActorInit En_Bji_01_InitVars = {
+                    case 0x5EE:
+                    case 0x5F2:
+                    case 0x5F5:
+                        func_80151938(globalCtx, globalCtx->msgCtx.unk11F04 + 1);
+                        break;
+                    case 0x5F0:
+                    case 0x5F6:
+                        func_80151938(globalCtx, 0x5EF);
+                        break;
+                    case 0x5E1:
+                    case 0x5E8:
+                    case 0x5E9:
+                    case 0x5EA:
+                    case 0x5EB:
+                    case 0x5F1:
+                    case 0x5F3:
+                    case 0x5F4:
+                    case 0x5F7:
+                    case 0x5F8:
+                        func_801477B4(globalCtx);
+                        this->actor.flags &= ~0x10000;;
+                        this->actor.params = ENBJI01_PARAMS_FINISHED_CONVERSATION;
+                        func_809CCE98(this, globalCtx);
+                        break;
+                }
+            }
+            break;
+        case 6:
+            this->actor.params = ENBJI01_PARAMS_FINISHED_CONVERSATION;
+            this->actor.flags &= ~0x10000;;
+            func_809CCE98(this, globalCtx);
+            break;
+    }
+    if ((this->animationIndex == 3) && (this->skelAnime.animCurrentFrame == this->skelAnime.animFrameCount)) {
+        func_8013E1C8(&this->skelAnime, D_809CDC7C, 2, &this->animationIndex);
+    }
+}
+
+void func_809CD634(EnBji01* this, GlobalContext* globalCtx) {
+    func_801A5BD0(0x6F);
+    func_801A89A8(0xE0000101);
+    globalCtx->nextEntranceIndex = 0x54A0; /* Termina Field from telescope */
+    gSaveContext.respawn[0].entranceIndex = globalCtx->nextEntranceIndex;
+    func_80169EFC(globalCtx); /* Load new entrance? */
+    gSaveContext.respawnFlag = -2;
+    this->actionFunc = EnBji01_DoNothing;
+}
+
+void EnBji01_DoNothing(EnBji01* this, GlobalContext* globalCtx) {
+}
+
+void func_809CD6C0(EnBji01* this, GlobalContext* globalCtx) {
+    func_8013E1C8(&this->skelAnime, D_809CDC7C, 2, &this->animationIndex);
+    this->actionFunc = func_809CD70C;
+}
+
+void func_809CD70C(EnBji01* this, GlobalContext* globalCtx) {
+    Math_ScaledStepToS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 0x444);
+    func_809CCDE0(this, globalCtx);
+    if (this->actor.shape.rot.y == this->actor.yawTowardsPlayer) {
+        func_800B86C8(&this->moonsTear->actor, globalCtx, &this->actor); /* Z-Target the Moon's Tear? */
+        this->actionFunc = func_809CD77C;
+    }
+}
+
+void func_809CD77C(EnBji01* this, GlobalContext* globalCtx) {
+    if (this->moonsTear->actor.colChkInfo.health == 0) {
+        func_809CCE98(this, globalCtx);
+    }
+}
+
+void EnBji01_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnBji01* this = THIS;
+
+    ActorShape_Init(&this->actor.shape, 0.0f, func_800B3FC0, 30.0f);
+    SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600578C, &D_06000FDC, this->jointTable, this->morphTable, 0x10);
```

Comment:
Now that you have a limb enum, you can use the LIMB_MAX here.
```suggestion
    SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600578C, &D_06000FDC, this->jointTable, this->morphTable, BJI_LIMB_MAX);
```